plugins/modules/proxmox_template.py

```diff
@@ -145,9 +145,9 @@
 
 class ProxmoxTemplateAnsible(ProxmoxAnsible):
     def get_template(self, node, storage, content_type, template):
```

Comment:
How about renaming this function to `has_template`? It's checking whether a template exists, and does not return it.

Reply:
Good call! Done!